includes/Framework/Api/JSONResponse.php

```diff
@@ -93,7 +93,7 @@ public function offsetExists( $offset ) {
 	 *
 	 * @return mixed The offset value.
 	 */
-	public function offsetGet( $offset ) {
+	public function offsetGet( $offset ): mixed {
```

Comment:
@layoutd I've been working on PHP 8.1 compatibility as well. But from my understanding the mixed return type was added in PHP 8.0. Looks like the minimum PHP version is still 7.2 here. Is that going to cause problems?

Reply:
Good catch, thanks @mikkamp. I looked over [this table](https://www.php.net/manual/en/language.types.declarations.php) while making the changes, that one snuck past me. Reverted in https://github.com/woocommerce/facebook-for-woocommerce/pull/2469. I'll have to loop back on a few of my recent changes to double-check them too.